support-frontend/assets/pages/[countryGroupId]/components/checkoutSummary.tsx

```diff
@@ -79,7 +79,11 @@ export default function CheckoutComponent({
 	const urlParams = new URLSearchParams(window.location.search);
 	const showBackButton = urlParams.get('backButton') !== 'false';
 
-	const isPaper = ['HomeDelivery', 'SubscriptionCard'].includes(productKey);
+	const isPaper = [
+		'HomeDelivery',
+		'NationalDelivery',
```

Comment:
I guess it was just an oversight from before that `NationalDelivery` wasn't included here? Or perhaps it's not an issue since `NationalDelivery` isn't ever the value of the `product` query string arg (it's transformed under the hood based on the postcode rather than passed explicitly).